Sources/TestsCommon/CodableExtensions.swift

```diff
@@ -174,3 +139,78 @@ extension MongoClientOptions: StrictDecodable {
         )
     }
 }
+
+extension ReadPreference: StrictDecodable {
```

Comment:
`StrictDecodable` was added to satisfy a unified test runner requirement - "the test runner MUST raise an error for an unsupported argument [for an operation]" - since without it we just wouldn't even try to decode a field if it were not something the options type supported. but it is a nice idea to use it generally for test-only decodable conformances to avoid silently ignoring fields 👍